docs/setup/independent/high-availability.md

```diff
@@ -197,9 +196,9 @@ In order to copy certs between machines, you must enable SSH access for `scp`.
 
     ```shell
     cfssl print-defaults csr > config.json
-    sed -i '0,/CN/{s/example\.net/'"$PEER_NAME"'/}' config.json
-    sed -i 's/www\.example\.net/'"$PRIVATE_IP"'/' config.json
-    sed -i 's/example\.net/'"$PUBLIC_IP"'/' config.json
+    sed -i -e '0,/CN/{s/example\.net/'"$PEER_NAME"'/}' \
```

Comment:
`config.json` was removed, which is rather important in this example... or it hasn’t been fulled updated to flow from the `cfssl print-defaults csr` inline. One or the other...

Reply:
@heckj 
agree with you